README.md

```diff
@@ -8,7 +8,9 @@ python time libraries mocking
 
```

Comment:
Line 7 (can't comment on it): CAN HAS POTATO.

Reply:
![image](https://user-images.githubusercontent.com/7465791/30554403-1613a968-9cad-11e7-86db-a8a90c1065bb.png)